process/transactions/dataFieldParser.go

```diff
@@ -0,0 +1,35 @@
+package transactions
+
+import (
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+const (
+	atSeparator = "@"
+	okHexConst  = "6f6b"
+)
+
+type argumentsParserExtended struct {
+	process.CallArgumentsParser
+}
+
+func newArgumentsParser(argsParser process.CallArgumentsParser) *argumentsParserExtended {
+	return &argumentsParserExtended{
+		CallArgumentsParser: argsParser,
+	}
+}
+
+func (age *argumentsParserExtended) split(dataField string) []string {
+	return strings.Split(dataField, atSeparator)
+}
+
+func (age *argumentsParserExtended) hasOKPrefix(dataField string) bool {
```

Comment:
ok is not quite the prefix as the prefix is the function name :D. Can be left as it is